src/sentry/static/sentry/app/components/narrowLayout.jsx

```diff
@@ -1,15 +1,41 @@
 import jQuery from 'jquery';
 import React from 'react';
+import PropTypes from 'prop-types';
+
+import {t} from 'app/locale';
+import {Client} from 'app/api';
+import styled from 'react-emotion';
+
+const Logout = styled.span`
+  float: right;
+  padding-top: 3px;
```

Comment:
same for this. if we used a flex layout this would automatically center this and the arbitrary margin wouldn't be necessary (and would scale up and down correctly)

Reply:
you could also wrap everything in a `Flex` component:

```js
<Flex alignItems="center">
  //header stuff
</Flex>